.openshift-ci/sensor_versions.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+"""
+Returns the latest patches of the last 4 major versions
+"""
+
+import sys
+import re
+import subprocess
+
+def isReleaseTagGitCLI(version):
+    return bool(re.search(r"^\d+\.\d+\.\d+$", version))
+
+def filterGitCLITags(rawtags):
+    filteredtags = []
+    for t in rawtags:
+        if isReleaseTagGitCLI(t):
+            filteredtags.append(t)
+    return set(filteredtags)
+
+def cliOutputToTags(stdoutput):
+    separated = stdoutput.decode(encoding="utf-8").splitlines()
+    return separated
```

Comment:
There's not much that this function does on its own. I suggest to inline it.

Reply:
Fair